22.09.2025/merge_sort.c

```diff
@@ -0,0 +1,57 @@
+#include "sorter.h"
+#include <stdio.h>
+#include <string.h>
+
+#define BUFFER_CAPACITY (100)
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof(arr[0]))
+
+/*
+ * Reads integers until newline in array 'arr'
+ * of size 'capacity', returns the number of integers
+ */
+int scanNumbers(int* arr, int capacity)
+{
+    char buf[BUFSIZ];
```

Comment:
```suggestion
    char buf[BUFSIZ] = { };
```

Нужно инициализировать и не только здесь

Reply:
Я же далее инициализирую массив с помощью fgets()